corpus/templates/components/navbar_large.html

```diff
@@ -1,6 +1,9 @@
 <div class="navbar bg-primary text-primary-content sticky top-0 z-10">
     <div class="navbar-start">
-        <a href="{% url 'index' %}" class="btn btn-ghost normal-case text-xl">Corpus</a>
+        <a href="{% url 'index' %}" class="btn btn-ghost normal-case text-xl">
+            <img src="../static/img/ieee-nitk.png" alt="IEEE NITK Logo" class="dark-logo w-20 h-10">
+            <img src="../static/img/ieee-nitk-white.png" alt="IEEE NITK Logo" class="light-logo w-20 h-10">
```

Comment:
Remove the `w-20` reference. This doesn't maintain the original aspect ratio, and instead squeezes the image into a 2:1 aspect ratio which is not what we want.

Reply:
fixed